latex_cite_completions.py

```diff
@@ -271,6 +271,23 @@ def _find_bib_files():
     return cache.LocalCache(root).cache('bib_files', _find_bib_files)
 
 
+def find_open_bib_files(active_window_only=False):
+    # get list of windows we want to check for valid buffers
+    if active_window_only:
+        windows = [sublime.active_window()]
+    else:
+        windows = sublime.windows()
+
+    # find which buffers are .bib files
+    result = []
+    for window in windows:
+        result += [v.file_name() for v in window.views() if v.file_name().endswith(".bib")]
```

Comment:
Change this to `if v.file_name() is not None and v.file_name().endswith(".bib")` or just to `if (v.file_name() or "").endswith(".bib")` to avoid an error, the view is not saved.